articles/hdinsight/hbase/apache-hbase-advisor.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Azure HDInsight Apache HBase Cluster/Query Advisor
+description: Gives a deeper understanding of the various Azure advisories for HDInsight HBase
+author: hrasheed-msft
+ms.author: ramvasu
+ms.reviewer: jasonh
+ms.service: hdinsight
+ms.topic: Cluster tuning
+ms.date: 12/02/2020
+
+#Customer intent: The azure advisories help to tune the cluster/query. This doc gives a much deeper understanding of the various advisories including the recommended configuration tunings.
+---
+
+# Reading most recent data
+If your use case happens to be reading the most recent data it means that most of the reads will land in memstore. The query advisory indicates that for a given column family in a table has > 75% reads that are getting served from memstore. This suggests that even if a flush happens on the memstore the recent file needs to be accessed and that needs to be in cache. To understand this even better, though the data is first written to memstore and we keep accessing the recent data, there is a chance that the flusher threads might that a given region has reached 128M (default) size and might trigger a flush. This specifically happens to the most recent data that was written when the memstore was around 128M in size. Such records might end up being a file read rather than from memstore. Hence it is better to ensure that even those data that are recently flushed can reside in the cache.
```

Comment:
If your use case reads the most recent data, there are high chances for the reads to be served from memstore, instead of the remote storage.